lib/node_modules/@stdlib/math/base/special/logf/README.md

```diff
@@ -66,17 +66,16 @@ v = logf( 2.0, -4.0 );
 <!-- eslint no-undef: "error" -->
 
 ```javascript
-var randu = require( '@stdlib/random/base/randu' );
-var roundf = require( '@stdlib/math/base/special/roundf' );
+var discreteUniform = require( '@stdlib/random/base/discrete-uniform' );
```

Comment:
@gunjjoshi Let's use `random/array/discrete-uniform` here and for `examples/index.js`.

Reply:
Done!